spec/polaris-management-service.yml

```diff
@@ -1054,8 +1054,15 @@ components:
               example: "https://s3.example.com:1234"
             stsEndpoint:
               type: string
-              description: endpoint for STS requests (optional). If not set, defaults to 'endpoint'.
+              description: endpoint for STS requests (optional). If not set, defaults to 'endpointInternal'.
               example: "https://sts.example.com:1234"
+            endpointInternal:
+              type: string
+              description: >-
+                endpoint for S3 requests made by the Polaris Server (optional). If set, Polaris Service will use
+                this value instead of "endpoint". If not set, defaults to 'endpoint'. Iceberg REST API clients never
+                see this value.
```

Comment:
Probably worth to update the description for `endpoint` as well, mentioning that there's `endpointInternal` in case the (client) needs to use a separate value than the service.

Reply:
updated